indico/modules/events/templates/display/_common.html

```diff
@@ -0,0 +1,74 @@
+{% macro render_location(item, parent=none, class='') -%}
+    {% set show_venue = not parent or (parent.venue_name != item.venue_name) %}
+    {% set show_room = item.room_name != '' %}
+    {% set show_url = item.room and item.room.map_url %}
+    <span class="icon-location item-location {{ class }}">
+        {%- if show_url %}
+            <a href="{{ item.room.map_url }}">
+        {%- endif -%}
+        {%- if show_venue and show_room -%}
+            {{ item.room_name }} ({{ item.venue_name }})
+        {%- elif show_venue -%}
+            {{ item.venue_name }}
+        {%- else -%}
+            {{ item.room_name }}
+        {%- endif -%}
+        {% if show_url -%}
+            </a>
+        {%- endif %}
+    </span>
+{%- endmacro %}
+
+{% macro render_users(user_list, span_class='', title=true, italic_affiliation=false, separator=', ', use_span=true) -%}
+    {% for link in user_list -%}
+        {% set user = link.person %}
+        {%- if use_span -%}
+            <span itemprop="performers" itemscope itemtype="http://schema.org/Person" class="{{ span_class }}">
+                {% if user.title -%}
+                    {{ user.title }}
+                {%- endif %}
+                <span itemprop="givenName">{{ user.first_name }}</span>
+                <span itemprop="lastName">{{ user.last_name }}</span>
+            </span>
+        {%- else -%}
+            {{ user.first_name }} {{ user.last_name }}
+        {% endif %}
+        {% if user.affiliation %}
+            <span class="affiliation">(
+            {%- if italic_affiliation -%}
```

Comment:
I think this would be more readable like this:

```
{%- if italic_affiliation -%}
    <em>{{ user.affiliation }}</em>
{%- else -%}
    {{ user.affiliation }}
{%- endif -%}
```